multimap/src/main/java/org/infinispan/multimap/impl/function/OfferFunction.java

```diff
@@ -0,0 +1,72 @@
+package org.infinispan.multimap.impl.function;
+
+import org.infinispan.commons.marshall.AdvancedExternalizer;
+import org.infinispan.functional.EntryView;
+import org.infinispan.multimap.impl.Bucket;
+import org.infinispan.multimap.impl.ExternalizerIds;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * Serializable function used by {@link org.infinispan.multimap.impl.EmbeddedMultimapListCache#offer(Object, Object)}
+ * to add a key/value pair.
+ *
+ * @author Katia Aresti - karesti@redhat.com
+ * @see <a href="http://infinispan.org/documentation/">Marshalling of Functions</a>
+ * @since 15.0
+ */
+public final class OfferFunction<K, V> implements BaseFunction<K, V, Void> {
+   public static final AdvancedExternalizer<OfferFunction> EXTERNALIZER = new Externalizer();
```

Comment:
The externalizer does not seem to be registered.

Reply:
added